indexers/gemini-3h/leaderboard/project.ts

```diff
@@ -161,7 +160,8 @@ const project: SubstrateProject = {
             kind: SubstrateHandlerKind.Block,
             handler: "handleSortAndRankLeaderboard",
             filter: {
-              modulo: BLOCK_SORT_AND_RANK_INTERVAL,
```

Comment:
Wouldn't be better to have this with the other constants?

Reply:
I opted to hard-code it since I removed all the other constants that I had put in src/mappings/constants.ts

Reply:
Okey, I saw It after my comment